tests/index.html

```diff
@@ -71,7 +71,7 @@
             image: true,
             imageUrlOrFunction: 'imageUrl',
             currency: '$',
-            header: '{{{checkInputQuery}}}',
+            header: 'Popular Products in {{hoverSuggestion}}',
```

Comment:
Can you add test cases for this
1. WIth simple string
2. with handlebar template
3. Without any value(empty string)